olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationCredentials.h

```diff
@@ -95,15 +95,6 @@ class AUTHENTICATION_API AuthenticationCredentials {
    */
   AuthenticationCredentials(std::string key, std::string secret);
```

Comment:
Mark as explicit?

Reply:
There is no need since there are 2 parameters already.